en/debug/precland.md

```diff
@@ -0,0 +1,47 @@
+# Precision Landing
+This document describes the working principle of the `landing_target_estimator` and the Precision Land flight mode.
+For setup and configuration instructions for precision landing with multicopters, refer to the [precision land tutorial](../tutorials/precland.html).
+
+## Landing Target Estimator
+The `landing_target_estimator` takes measurements from the `irlock` driver (`irlock_report`s) as well the estimated terrain height (`vehicle_local_position.dist_bottom`) to estimate the beacon's position relative to the vehicle.
+
+The measurements in `irlock_report` contain the tangent of the angles from the image center to the beacon. In other words, the measurements are the x and y components of the vector pointing towards the beacon, where the z component has length 1.
+This means that scaling the measurement by the distance from the camera to the beacon results in the vector from the camera to the beacon.
+This relative position is then rotated into the north-aligned, level body frame using the vehicle's attitude estimate.
+Both x and y components of the relative position measurement are filtered in separate Kalman Filters, which act as simple low-pass filters that also produce a velocity estimate and allow for outlier rejection.
+
+The `landing_target_estimator` publishes the estimated relative position and velocity, whenever a new `irlock_report` is fused into the estimate. Nothing is published if the beacon is not seen or beacon measurements are rejected.
+The landing target estimate is published in the `landing_target_pose` message.
+
+
+## Vehicle state estimate aiding
+> **Note** Vehicle state estimate aiding is currently only supported with LPE.
+
+If the beacon is specified to be stationary using the parameter `LTEST_MODE`, the vehicle's position/velocity estimate can be improved with the help of the beacon measurements.
+This is done by fusing the beacon's velocity as a measurement of the negative velocity of the vehicle.
+
+## Precision Land controller
+A precision landing can be initiated by switching to the Precision land flight mode:
```

Comment:
Can QGC be configured to assign this mode to a switch yet?

Reply:
No, this is currently not possible.